packages/ide-toolbox/src/error-boundary/error-boundary.module.scss

```diff
@@ -1,6 +1,17 @@
-.errorMessage {
-  text-align: center;
+.errorMessageContainer {
+  width: 100%;
+  height: 100%;
   padding-left: 20px;
   padding-right: 20px;
-  color: red;
 }
+
+.errorMessageTitle {
+  color: #d32f2f;
```

Comment:
Let's set background color also, to make sure text is readable (it can be different in the app)